sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 1.0.0b23 (Unreleased)
```

Comment:
After releases, please make sure to appr and merge the automatically created post-rlease prs. Then you wont have to do these manually. https://github.com/Azure/azure-sdk-for-python/pull/34198

Reply:
That [task failed](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3471076&view=logs&j=9e408172-77a7-548b-8cd1-e45278a8c989&t=e65141db-c30e-5b82-93af-081ba0717a0b), so I assumed the PR wasn't created. Strange.